pkg/apis/policy/validation/validation.go

```diff
@@ -78,6 +80,17 @@ func ValidatePodDisruptionBudgetSpec(spec policy.PodDisruptionBudgetSpec, opts P
 
 	allErrs = append(allErrs, unversionedvalidation.ValidateLabelSelector(spec.Selector, labelSelectorValidationOptions, fldPath.Child("selector"))...)
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.PDBPodHealthyPolicy) {
+		supportedPolicies := sets.NewString(
```

Comment:
It's reasonable to move this var outside the function body, to not to instantiate it every time.